pkg/auth/token_test.go

```diff
@@ -9,6 +9,22 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const sampleRSAPrivateKey = `-----BEGIN RSA PRIVATE KEY-----
+MIICXwIBAAKBgQDN+wofEP/rjz8JaZz9vjuplDRoIMH0fEh2BMcbEkRLY9lbhNi/
+w0if+ytwRkH7i4thX43IQ4+EhlmbgIzzUXQfXdAct2vNbiiFy0FyNIlz67Yknt2q
+uHf2RgKGXu/vcMI7Dguyajo5mHwLiOoJ7ql86uNtquLiTvE5eLqd1gWRFQIDAQAB
+AoGBALeT2qRveTdPFsZjy1hWuEPd44s+Tr6AGfCdN3rIH/f1CJ5JWwglms+Cgmdx
+JpNy/gkNqYZnuDxLpQczXevpl4xXs5evn3mpeP668zLyzx0u6FJkS905MvOKF4vk
+O+4eAa/12dpV7vEpWiLtZc3n+h4Y4L7EEluGR6VvwEbS/5cNAkEA5t5lhgL+x6u6
+++hLOXZHg3bU7V5HfIdsoX8lAeZwgpvrcWzllZYoO7PQWaITM+ibcpjKQNCM3Jnx
+iDBkfuW5fwJBAORnFmxkwZXOxfH4DwK+oup9x34RVMHXI9Y12xy5MOgKb5vgRDeN
+el7m9DkyRYRo8TM4633gmTrFAx9Gogq5d2sCQQC6AqfjwJgMwmWmPzQUuSLHXkAS
+e+q2/9nbiLiFfmhaI0wgmC+mRVRnPep5vWchZKGSRF54uE82EmaTZwIhZ+/7AkEA
+sTXKkA8co77qlfKAswB2JrmwLoAD4uGpTGo8tux4pZBzR92ZEAEVEMzgcAAxL6q8
+eaGQFPpN6OsyoPGMiAWeQQJBAIJjzp5vKxYAA6OZuc9h7mTChqGZWE8xWhT0qUxY
+/nDqqxFYcBZ8BzD+Os3MF+Vnvewqmh4vZP69t8mVnuIF6Do=
+-----END RSA PRIVATE KEY-----`
```

Comment:
Doesn't this trigger every secrets detector?

**ALSO**, ouch, that looks like PKCS#1.  Can we use something from the current millennium such as PKCS #8 ("BEGIN PRIVATE KEY" without the "RSA"; RSA gets encoded in the key itself)?  If we did, then even the [Wikipedia page](https://en.wikipedia.org/wiki/PKCS_8) contains a sample private key, which should be safe to use and include in a repository.

Reply:
If we must use this, please include the command-line which generated it.  (It is OK for it to be nondeterministic and generate a different key each time.  Just please don't leave it mysterious how to generate a new one and/or its source!)